components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -51,6 +51,10 @@
                         ],
                         "deprecationMessage": "The 'protocol' property is deprecated.",
                         "description": "The protocol to be used. (deprecated)"
+                    },
```

Comment:
as discussed in https://github.com/gitpod-io/gitpod/issues/3059#issuecomment-984607394 description is helpful but it is hidden so it would be good to reintroduce name attribute as well, which will be used as a prefix in the remote explorer.

Reply:
Got it, I'll adjust and reintroduce it soon.

Reply:
I've reintroduced the port name attribute, and it's now being provided through the PortStatus, along with the description.